packages/wire-cell-toolkit/package.py

```diff
@@ -218,5 +218,8 @@ def install(self, spec, prefix):
         python(*ins)
 
     def setup_run_environment(self, env):
-        env.prepend_path("WIRECELL_PATH", self.prefix.share.wirecell);
+        env.prepend_path("WIRECELL_PATH", self.prefix.share.wirecell)        
```

Comment:
Setting `WIRECELL_PATH` by default is a bit controversial. Here's the issue:

Most users/experiments maintain their configs separate from their WCT release.  They include the subset of top level files that get installed into `share/wirecell/`.  Setting `WIRECELL_PATH` here is incomplete and then the user must make sure to **prepend** their real cfg area to assure any updates that they provide take precedence over those in `share/wirecell/`.  If they instead append, they can get hard to find bugs.

I know FNAL UPS packages do set this env var.  I think that is also a net, if a small, badness.

Reply:
Thanks, that is good information and I didn't know that was an issue. I wasn't intending to change the behavior of WIRECELL_PATH in this line, I was just removing a semicolon which I thought was unnecessary. Maybe I am underestimating the role of that semi-colon in the problem you described?

Reply:
Oh!  Duh, sorry.  I misread the diff.  Yeah, the semicolon is wrong, if only cosmetically.

I suggest we let this PR stew for a while and if we find out there is some benefit from adjusting the `LD_LIBRARY_PATH` we can revive things.